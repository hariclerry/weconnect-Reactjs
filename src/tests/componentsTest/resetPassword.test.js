import React from "react";
import {shallow, mount} from "enzyme";
import { MemoryRouter } from "react-router-dom";
import jwt from "jsonwebtoken";
import weconnectStore from "store";
import ResetPassword from "containers/user/resetPassword";

const loginUserMock = { access_token: jwt.sign({ email: "clerry@gmail.com", password: "this199" }, "hard to guess string")}

describe('ResetPassword component',() =>{
  beforeEach(() => {
    localStorage.setItem("token", loginUserMock.access_token);
});
it('should match snapshot',() =>{
  const wrapper = shallow(<ResetPassword />)
  expect(wrapper).toMatchSnapshot();
})

it("should render the ResetPassword component", () => {
  const mountComponent = shallow(<MemoryRouter>< ResetPassword store = {weconnectStore}/></MemoryRouter>)
  expect(mountComponent.length).toBe(1);
  
});

}

)