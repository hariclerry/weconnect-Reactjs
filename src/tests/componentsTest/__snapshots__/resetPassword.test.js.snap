// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResetPassword component should match snapshot 1`] = `<Route />`;
